csharp/Platform.Setters/SetterBase.cs

```diff
@@ -6,47 +6,57 @@
 namespace Platform.Setters
 {
     /// <summary>
-    /// <para>Represents a base implementation for an setter that allows you to set a passed value as the result value.</para>
-    /// <para>Представляет базовую реализацию для установщика, который позволяет установить переданное ему значение в качестве результирующего значения.</para>
+    /// <para>Provides a base class that allows to set a passed value as the result value.</para>
+    /// <para>Предоставляет базовый класс, который позволяет установить переданное ему значение в качестве результирующего значения.</para>
     /// </summary>
-    /// <typeparam name="TResult"><para>The type of result value.</para><para>Тип результирующего значения.</para></typeparam>
+    /// <typeparam name="TResult">
+    /// <para>The type of a result value.</para>
+    /// <para>Тип результирующего значения.</para>
+    /// </typeparam>
     /// <remarks>
-    /// Must be class, not struct (in order to persist access to Result property value).
+    /// <para>Must be class, not struct (in order to persist access to Result property value).</para>
+    /// <para>Является классом, не структурой (чтобы сохранить доступ к значению свойства Result).</para>
     /// </remarks>
     public abstract class SetterBase<TResult> : ISetter<TResult>
     {
         /// <summary>
-        /// <para>Represents the result value.</para>
+        /// <para>Represents a result value.</para>
         /// <para>Представляет результирующие значение.</para>
         /// </summary>
         protected TResult _result;
         
         /// <summary>
-        /// <para>Gets result value.</para>
+        /// <para>Gets a result value.</para>
         /// <para>Возвращает результирующее значение.</para>
         /// </summary>
         public TResult Result => _result;
         
         /// <summary>
-        /// <para>Initializes a new instance of the SetterBase class.</para>
-        /// <para>Инициализирует новый экземпляр класса SetterBase.</para>
+        /// <para>Initializes a new instance of the <see cref="SetterBase{TResult}"/> class.</para>
```

Comment:
```suggestion
        /// <para>Initializes a new instance of the <see cref="SetterBase"/> class.</para>
```
There is no type parameters